server/src/main/java/io/unitycatalog/server/persist/dao/StagingTableDAO.java

```diff
@@ -0,0 +1,76 @@
+package io.unitycatalog.server.persist.dao;
+
+import io.unitycatalog.server.model.StagingTableInfo;
+import jakarta.persistence.*;
```

Comment:
Maybe I need to enforce the checkstyle rules so that we won't introduce more `*` package here.